### ISSUE TITLE
Reproducing results for SimCLR on tiny-imagenet

### ISSUE THREAD

**Issue by aakashrkaku** [htdt/self-supervised#6]
Hi Authors,

I was trying to reproduce the results (in the paper) of the SimCLR model for the Tiny-imagenet dataset. I used default training/hyper-parameters but I was not able to achieve results that were close to the ones reported in the paper. Specifically, I cloned this repository and then ran the following command (without changing any default hyper-parameters): `python -m train --dataset tiny_in --epoch 1000 --lr 2e-3 --emb 128 --method contrastive`

The model hardly reached a top1 accuracy of 1% after 500 epochs. Can you please let me know if I need to change some default hyper-parameters / code to be able to reproduce the results? Thank you!

Kind regards,
Aakash

**Comment by htdt** [comment]
I would guess that the dataset was not set up properly. You need a specific pytorch folder structure. As far as I remember, [this repo](https://github.com/tjmoon0104/pytorch-tiny-imagenet) includes a script to prepare it.
